WrathCombo/CustomCombo/Functions/Target.cs

```diff
@@ -571,8 +571,12 @@ internal unsafe static bool IsInLineOfSight(IGameObject target)
             return isLoSBlocked == false;
         }
 
-        internal unsafe static bool IsQuestMob(IGameObject target) => target.Struct()->NamePlateIconId is 71204 or 71144 or 71224 or 71344;
+        internal static unsafe bool IsQuestMob(IGameObject target) => target.Struct()->NamePlateIconId is 71204 or 71144 or 71224 or 71344;
 
-        internal unsafe static bool IsBoss(IGameObject target) => Svc.Data.GetExcelSheet<BNpcBase>()?.GetRow(target.DataId).Rank is 2 or 6;
+        internal static bool IsBoss(IGameObject? target) => Svc.Data.GetExcelSheet<BNpcBase>().GetRow(target.DataId).Rank is 2 or 6;
```

Comment:
If we're adding the property `TargetIsBoss`, I think this method ought to be private.